### PR TITLE
Welcome: Add open science philosophy documentation

### PR DESCRIPTION
**Pull request by kairstenfay**
Add a quick blurb about the SFS open science / open source philsophy
along with a quick guide to what's ok vs. not ok to share publicly.

This felt sorely needed after today's lengthy [Slack conversation](https://seattle-flu-study.slack.com/archives/C0173B4S1LY/p1594767049432900) started by @jccraft-uw. I tried my best to summarize it here. Please let me know if I missed anything!

### CODE REVIEW

**Comment by kairstenfay** [comment]
I just pushed a small copy-edit fixup.

File: Welcome.md [hunk 1]
```diff
@@ -44,6 +47,33 @@ start using or contributing to our code.
...
+### Safe to share publicly
+* Documentation that is Seattle Flu specific but not sensitive
+* Links to Slack conversations
+* Links to Trello cards
+* Links to Metabase (or other apps at backoffice.seattleflu.org)
+* First names of study contacts
+
+
+### Don't share publicly
+* Personally identifiable information of study participants are employees
```
Comment (jccraft-uw) [inline Welcome.md:66]: replace "are" with "or"

File: Welcome.md [hunk 2]
```diff
@@ -44,6 +47,33 @@ start using or contributing to our code.
     - [Prevent wrapping within psql (optional)](#prevent-wrapping-within-psql-optional)
 
 
+## Open source philosophy
+The Seattle Flu Study practices open science.
+This means that our code (with very few, notable exceptions) is all open source.
```
Comment (jccraft-uw) [inline Welcome.md:52]: suggestion to include documentation too:
This means that our code and documentation (with very few, notable exceptions) are all open source.

File: Welcome.md (hunk 1 above)
Comment (kairstenfay) [reply Welcome.md:66]: Oops! Meant to push that in the fixup commit, but I missed it. I'll push it soon then rebase before merging.

**Comment by kairstenfay** [comment]
All changes are fixed up into one commit. Thanks all for your contributions!

**Pull request merged**